---
title: "What's in Foundation Medicine's "Secret Sauce"?"
---

Forbes reports on <a href="http://www.forbes.com/sites/matthewherper/2013/01/08/bill-gates-invests-in-cancer-dna-sequencing-firm/?goback=.gde_1902623_member_202645735">Bill Gates recent investment in Foundation Medicine</a>:

> Founded in April 2010 by Third Rock Ventures, Foundation is one of  the first companies to aim to take advantage of the explosion in DNA  sequencing technology. ... It now costs as little as  $1,000 to get a fairly accurate readout of the 6 billion letters of DNA  code for any single person.
In cancer, the approach right now is usually not to sequence all a  patient’s DNA or that of his tumor, but instead to focus on particular  genetic mutations in the tumor that might provide clues as to what  medicines to try. Major cancer centers are using this approach with  patients for whom it’s not obvious which medicine represents the best  bet. Foundation’s approach has been to provide that kind of testing to a  larger audience.

Without a doubt, [Foundation Medicine](http://www.foundationmedicine.com/) is one company that's translating basic research in a big way. Here's how:
[]()Their main product, FoundationOne, is a targeted sequencing assay that covers 236 genes as well as regions of 19 genes that have been reported in DNA rearrangements, which is a frequent event in cancerous cells. [This poster](http://www.foundationone.com/about-foundationone/ONE-B-001-20130115_FoundationOne_Technical.pdf) describes these genes and regions in much more detail. If you're interested in the details of how the test does what it's intended to do, a patent application submitted last year for "<a href="http://appft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&amp;Sect2=HITOFF&amp;p=1&amp;u=%2Fnetahtml%2FPTO%2Fsearch-bool.html&amp;r=1&amp;f=G&amp;l=50&amp;co1=AND&amp;d=PG01&amp;s1=multigene.TTL.&amp;s2=analysis.TTL.&amp;OS=TTL/multigene+AND+TTL/analysis&amp;RS=TTL/multigene+AND+TTL/analysis">Optimization of multigene analysis of tumor samples</a>", which tries to cover almost any strategy you can think of that can be used to detect one or more of the genes in the test.

The basic approach for the test is a standard one: isolate DNA from a tumour sample, enrich for DNA that you're interested in (as other 'uninteresting' regions of the genome introduce noise in the data being analyzed), and sequence, in this case using one of Illumina's machines. The DNA enrichment outlined uses Agilent's <a href="http://www.genomics.agilent.com/CollectionOverview.aspx?PageType=Application&amp;SubPageType=ApplicationOverview&amp;PageID=2094">SureSelect DNA target enrichment system</a> as the technology that this product is based on. Agilent has a <a href="http://www.genomics.agilent.com/GenericB.aspx?pagetype=Custom&amp;subpagetype=Custom&amp;pageid=3083">basic walkthrough</a> of how the technology works. So, if you're interested in getting on the personalized medicine bandwagon and you're not Bill Gates or an investor in Third Rock, you might want to consider companies like Illumina or Agilent which sell the infrastructure that will be used to enable all these tests.

The company does face challenges. Last year, the MIT Technology Review published a great <a href="http://www.technologyreview.com/featuredstory/426987/foundation-medicine-personalizing-cancer-drugs/?mod=chfeatured">description of the headwinds facing Foundation</a>, with emphasis on a potential patent thicket on key genes in the test, and more importantly how quickly the test will be adopted by users, namely physicians, which aren't usually being trained to deal with the details in these kinds of reports:

> Another obstacle is that the idea of using DNA to guide cancer  treatment puts doctors in an unfamiliar position. Physicians, as well as  the FDA and insurance companies, still classify tumors and drug  treatments anatomically. “We’re used to calling cancers breast, colon,  salivary,” says oncologist Thomas Davis, of the Dartmouth-Hitchcock  Medical Center, in Lebanon, New Hampshire. “That was our shorthand for  what to do, based on empirical experience: ‘We tried this drug in  salivary [gland] cancer and it didn’t work.’ ‘We tried this one and 20  percent of the patients responded.’” 
Now the familiar taxonomy is  being replaced by a molecular one. It was Davis who ordered DNA tests  from several companies for the patient with the salivary-gland tumor. “I  got bowled over by the amount of very precise, specific molecular  information,” he says.

One final, but hugely interesting point you can gather from all of [Foundation's publications](http://www.foundationmedicine.com/diagnostics-publications.php) is that they're top tier, from an academic perspective. The paper on [colorectal and lung cancer gene fusions](http://www.nature.com/nm/journal/v18/n3/abs/nm.2673.html) went to *Nature*, and another one on [pediatric lymphoma](http://bloodjournal.hematologylibrary.org/content/120/26/5181.abstract) was published in *Blood*. Both papers had key people on them that are affiliated with the company, so if you're a PhD student, don't let a supervisor convince you that academic research is the only place to do great work. 




